scrapers/SARJ-LLC.py

```diff
@@ -50,7 +94,21 @@ def fetch(baseUrl, type, arguments):
     return data
 
 def scrape_model(baseUrl, name):
-    data = fetch(baseUrl, 'model', {'name': name})
+    transformed_name = str.join(
+      ' ',
+      list(
+        map(
+          lambda p:
+            re.sub(
+              '[_-]',
+              ' ',
+              re.sub('\w\S*', lambda m: m.group(0).lower().capitalize(), p),
+            ),
+          name.split('-')
+        )
+      )
+    )
```

Comment:
This logic is converted from the JavaScript of the website. In short it capitalizes every part of the name (splitted on the `-`) and joins the parts with a space. So URL `/model/foo-bar` would become `/api/model?name=Foo%20Bar`. Without this (and adding `order` and `direction`) sometimes calls would result in a model not found error. With these 2 changes it seems to work fine every time.

Reply:
i noticed the search was acting weird good catch